lib/pbench/test/functional/server/test_audit.py

```diff
@@ -0,0 +1,44 @@
+from pbench.client import API, PbenchServerClient
+
+
+class TestAudit:
+    def test_get_all(self, server_client: PbenchServerClient, login_admin):
+        """
+        Verify that we can retrieve the Pbench Server audit log.
+
+        This relies on a "testadmin" user which has been granted ADMIN role
+        via the pbench-server.cfg file for functional testing. The audit API
+        should succeed without permissions failure, and we'll validate the
+        audit fields of the records we see.
+        """
+        response = server_client.get(API.SERVER_AUDIT, {})
+        json = response.json()
+        assert (
+            response.ok
+        ), f"Reading audit log failed {response.status_code},{json['message']}"
+        assert isinstance(json, list)
```

Comment:
So, `json` is a `dict` on failure and a `list` on success?  😝

Reply:
Our error reporting convention has always been a `{"message": "<reason>"}` JSON object, but on success each API has its own standard behavior. Yeah, in theory we could force them all to return a JSON object anyway, but while I've considered that it seems forced and unhelpful.

Then again, it might not be a bad idea anyway in this case since I *do* need to add pagination here, where we'll likely end up with a JSON object like `GET /datasets` anyway. Not appropriate for *this* PR, however.